src/calculateRentalCost.js

```diff
@@ -28,7 +28,24 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayPrice = 40;
+  let fullPrice = 0;
+  const minDiscount = 20;
+  const maxDiscount = 50;
+
+  if (days <= 2) {
+    fullPrice = dayPrice * days;
```

Comment:
as you can see over your code, `total` always has its default value of `dayPrice * days`, so you put it there right away